src/modules/sectionMethods.ts

```diff
@@ -229,14 +256,18 @@ const update = async (data: UpdateSection): Promise<Section> => {
         attaches: attachIds ? { connect: idsToIdObjs(attachIds) } : undefined,
     };
 
-    sendHrMail && notifyHR(sectionId, data);
-
     const updatedSection = await prisma.section.update({
         data: updateData,
         where: { id: sectionId },
         include: { sectionType: true },
     });
 
+    if (sendHrMail) {
+        await notifyHR(sectionId, data);
+
+        await giveSectionAchievement(interviewerId);
```

Comment:
why this is under sendHrMail flag?

Reply:
This flag became more universal, I'll rename it